helm/values.yaml

```diff
@@ -216,1462 +217,1489 @@ smtp:
+            request: my_policy ## ILM policy for the gravitee-request-* indexes
+            health: my_policy ## ILM policy for the gravitee-health-* indexes
+            log: my_policy ## ILM policy for the gravitee-log-* indexes
+        # http:
+        # timeout: 10000
+    ssl:
+        enabled: false
+        # keystore:
+        #   type: jks
+        #   path: path/to/jks
+        #   password: example
+        #   certs:
+        #     - /path/to/cert1
+        #     - /path/to/cert2
+        #   keys:
+        #     - /path/to/key
+        #     - /path/to/key2
+    endpoints:
+        - http://graviteeio-apim-elasticsearch-ingest-hl:9200
+    pipeline:
+        plugins:
+            ingest: geoip, user_agent # geoip and user_agent plugins are enabled by default
 
 elasticsearch:
-  enabled: false
-  name: "elasticsearch"
-  fullnameOverride: "graviteeio-apim-elasticsearch"
-  image:
-    repository: "bitnami/elasticsearch"
-    tag: "8.8.2"
-  master:
-    replicas: 2
-    persistence:
-      size: 4Gi
-    resources:
-      limits:
-        cpu: 500m
-        memory: 1024Mi
-      requests:
-        cpu: 25m
-        memory: 256Mi
-  coordinating:
-    replicas: 2
-    resources:
-      limits:
-        cpu: 500m
-        memory: 1024Mi
-      requests:
-        cpu: 25m
-        memory: 256Mi
-  data:
-    replicas: 2
-    heapSize: 512m
-    resources:
-      limits:
-        cpu: 1
-        memory: 2048Mi
-      requests:
-        cpu: 50m
-        memory: 1024Mi
-    persistence:
-      size: 20Gi
+    enabled: false
+    name: "elasticsearch"
+    fullnameOverride: "graviteeio-apim-elasticsearch"
+    image:
+        repository: "bitnami/elasticsearch"
+        tag: "8.8.2"
+    master:
+        replicas: 2
+        persistence:
+            size: 4Gi
+        resources:
+            limits:
+                cpu: 500m
+                memory: 1024Mi
+            requests:
+                cpu: 25m
+                memory: 256Mi
+    coordinating:
+        replicas: 2
+        resources:
+            limits:
+                cpu: 500m
+                memory: 1024Mi
+            requests:
+                cpu: 25m
+                memory: 256Mi
+    data:
+        replicas: 2
+        heapSize: 512m
+        resources:
+            limits:
+                cpu: 1
+                memory: 2048Mi
+            requests:
+                cpu: 50m
+                memory: 1024Mi
+        persistence:
+            size: 20Gi
 
 alerts:
-  enabled: false
-  endpoints:
-    - http://localhost:8072/
-  security:
     enabled: false
-    username: admin
-    password: adminadmin
+    endpoints:
+        - http://localhost:8072/
+    security:
+        enabled: false
+        username: admin
+        password: adminadmin
 
 management:
-  type: mongodb
-  #url: your_public_ui_url
+    type: mongodb
+    #url: your_public_ui_url
 
 ratelimit:
-  type: mongodb
+    type: mongodb
```

Comment:
Is there any reason to have mongodb as default? Maybe it would be better to switch that to redis?

Reply:
I didn't change this.
Don't look at the reformatting commit (first one) then you will see what has changed.